apps/deploy-web/src/utils/walletUtils.ts

```diff
@@ -7,12 +7,7 @@ export type LocalWalletDataType = {
   name: string;
   selected: boolean;
   isManaged?: boolean;
-};
-
-export const useStorageWallets = () => {
-  const wallets = getStorageWallets();
-
-  return { wallets };
+  userId?: string;
```

Comment:
Maybe I missed it, where do we set this?

Reply:
when managed wallet is fetched from the backend it has a userId and is persisted as is to the local storage